config/environments/test.rb

```diff
@@ -18,4 +18,6 @@
   config.action_view.raise_on_missing_translations = true
   config.action_mailer.default_url_options = { host: "www.example.com" }
   config.active_job.queue_adapter = :inline
+
+  config.aact = {static_files_directory: "public/static/aact-files"}
```

Comment:
This should be changed for tests. We don't want to store temporary test files in the directory with all the other files.

Change the directory to use the tmp folder 'tmp/aact-files'